packages/wdio-static-server-service/src/@types/wdio-logger-stub.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@wdio/logger';
```

Comment:
This is not needed anymore, please rebase on top of latest master and remove this file

Reply:
Done